zipline/data/minute_bars.py

```diff
@@ -587,6 +585,78 @@ def _get_metadata(self):
     def first_trading_day(self):
         return self._first_trading_day
 
+    def _minutes_to_exclude(self):
+        """
+        Calculate the minutes which should be excluded when a window
+        occurs on days which had an early close, i.e. days where the close
+        based on the regular period of minutes per day and the market close
+        do not match.
+
+        Returns:
+        --------
+        List of DatetimeIndex representing the minutes to exclude because
+        of early closes.
+        """
+        market_opens = self._market_opens.values.astype('datetime64[m]')
+        market_closes = self._market_closes.values.astype('datetime64[m]')
+        minutes_per_day = (
+            self._market_closes.values.astype('datetime64[m]')
+            -
+            self._market_opens.values.astype('datetime64[m]')
```

Comment:
Good catch, will change over to using those. (Had intended to switch over to those during a refactoring.)